core/lib/generators/solidus/install/frontend_templates/solidus_frontend.rb

```diff
@@ -0,0 +1,3 @@
+apply "#{__dir__}/break_down_solidus_gem.rb"
```

Comment:
That changes the current behavior. We weren't breaking down the solidus meta-gem when solidus_frontend was chosen, as there's not actual need. Am I missing something?

Reply:
That's true, I think I did it for parity with the others but it's fine to keep the meta-gem intact in this case, I'll update the template 👍

Reply:
Thanks!